src/theme/components/componentWrapper.ts

```diff
@@ -5,6 +5,7 @@ interface ComponentWrapperProps {
 const ComponentWrapper = {
   parts: ["helperErrorText"],
   baseStyle: ({ hasChildren }: ComponentWrapperProps) => ({
+    width: "fit-content",
```

Comment:
@EdwinGuzman Yes, this is causing a problem with any component that is supposed to expand to full width by default.  This would include:

- `Checkbox` and `CheckboxGroup`
- `Radio` and `RadioGroup`
- `SearchBar`
- `Select`
- `Slider`
- `TextInput`
- `Toggle`
- `VideoPlayer`

Reply:
Okay, how about we do this:
1) @jackiequach removes this `width: "fit-content"` rule in this PR.
2) We create a new PR where we add a `width` prop to the `ComponentWrapper` so that the `Tooltip` can render correctly.

Now that I'm thinking about this, maybe you can remove this rule from this component _but_ pass it as a style prop. So in `Tooltip.tsx`:
```
    const newChildren = shouldWrapChildren ? (
      <ComponentWrapper width="fit-content">{children}</ComponentWrapper>
    ) : (
      children
    );
```

This should work and makes sense to do it _only_ for this use-case with the tooltip. This might be a better approach than creating a new PR. @jackiequach let us know if this works.

Reply:
Passing as a style prop works. Updated.